tests/e2e/plugins/module-setup-analytics-no-account.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Plugin Name: E2E Tests Module Setup Analytics API Mock No Account
+ * Plugin URI:  https://github.com/google/site-kit-wp
+ * Description: Utility plugin for mocking Analytics Setup API requests during E2E tests.
+ * Author:      Google
+ * Author URI:  https://opensource.google.com
+ */
```

Comment:
Might wanna add the license info here, like #416 did for the other utility plugins already.

Reply:
Added